regression-test/data/nereids_syntax_p0/function.out

```diff
@@ -11,6 +11,11 @@
 -- !count --
 3	3
 
+-- !distinct_count_with_group_by --
+1	1
+1	1
+1	1
```

Comment:
we need to load more data to table to verify it

Reply:
Data cannot be easily added to the existing test set...